build_runner/README.md

```diff
@@ -110,12 +185,55 @@ you can type `n` to abandon the build without taking any action.
 
 ### Publishing packages
 
-In general generated files should be published with your package, but this may
-not always be the case. Some `Builder`s may provide a recommendation for this as
-well.
+In general generated files **should** be published with your package, but this
+may not always be the case. Some `Builder`s may provide a recommendation for
+this as well.
+
+## Contributing
+
+We welcome a diverse set of contributions, including, but not limited to:
+
+* [Filing bugs and feature requests][file_an_issue]
+* [Send a pull request][pull_request]
+* Or, create something awesome using this API and share with us and others!
+
+For the stability of the API and existing users, consider opening an issue
+first before implementing a large new feature or breaking an API. For smaller
+changes (like documentation, minor bug fixes), just send a pull request.
+
+### Testing
```

Comment:
Highlight that we have travis enabled – so pull requests will be validated for all of these.

Reply:
Done.